src/Tickets/Emails/Email/Purchase_Receipt.php

```diff
@@ -0,0 +1,340 @@
+	 *
+	 * @var string
+	 */
+	public $template = 'customer-purchase-receipt';
+
+	/**
+	 * Email recipient.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public $recipient = 'customer';
+
+	/**
+	 * Enabled option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_enabled = 'tec-tickets-emails-purchase-receipt-enabled';
+
+	/**
+	 * Subject option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_subject = 'tec-tickets-emails-purchase-receipt-subject';
+
+	/**
+	 * Email heading option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_heading = 'tec-tickets-emails-purchase-receipt-heading';
+
+	/**
+	 * Email heading option key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static $option_add_content = 'tec-tickets-emails-purchase-receipt-add-content';
+
+	/**
+	 * Checks if this email is enabled.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_enabled(): bool {
+		return tribe_is_truthy( tribe_get_option( static::$option_enabled, true ) );
+	}
+
+	/**
+	 * Checks if this email is sent to customer.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_customer_email(): bool {
+		return true;
+	}
+
+	/**
+	 * Get email title.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email title.
+	 */
+	public function get_title(): string {
+		return esc_html__( 'Purchase Receipt', 'event-tickets' );
+	}
+
+	/**
+	 * Get email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email heading.
+	 */
+	public function get_heading(): string {
+		$heading = tribe_get_option( static::$option_heading, $this->get_default_heading() );
+
+		/**
+		 * Allow filtering the email heading for Purchase Receipt.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_purchase_receipt_heading', $heading, self::$id, $this->template );
+
+		/**
+		 * Allow filtering the email heading globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string          $heading  The email heading.
+		 * @param string          $id       The email id.
+		 * @param Tribe__Template $template Template object.
+		 */
+		$heading = apply_filters( 'tec_tickets_emails_heading', $heading, self::$id, $this->template );
+
+		return $this->format_string( $heading );
+	}
+
+	/**
+	 * Get default email heading.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public function get_default_heading() {
+		return esc_html__( 'Your purchase receipt for #{order-number}', 'event-tickets' );
+	}
+
+	/**
+	 * Get email subject.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The email subject.
+	 */
+	public function get_subject(): string {
+		$subject = tribe_get_option( static::$option_subject, $this->get_default_subject() );
+
+		$subject = $this->format_string( $subject );
```

Comment:
We should do the format string later, as the last step, so that there's access to the placeholders when they filter.